snippets/vhdl.yml

```
@@ -163,3 +217,9 @@ zeroes:
   description: Zero Others
   scope: source.vhdl
   body: "others => '0'"
+
+integer_range_limitation:
+  prefix: intr
+  description: Integer (Range Limitation)
+  scope: source.vhdl
+  body: "integer range ${1:L1} ${2|downto,to|} ${3:L2}"
```

Good suggestion, however I would prefer if the default suggestion was valid integers.
```suggestion
  body: "integer range ${1:0} ${2|downto,to|} ${3:255}"
```